_inc/client/components/product-expiration/index.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { moment, translate as __ } from 'i18n-calypso';
+
+class ProductExpiration extends React.PureComponent {
+	static propTypes = {
+		expiry_date: PropTypes.string,
```

Comment:
Something I missed to suggest in my previous review: since this is a brand new component, let's follow the variable name best practices and rename all these props to be `camelCase`.

Reply:
Resolved with 42e250c2a527d9eb7ad4b9c1140caa3106dcbaa4